docs/rules/max-attributes-per-line.md

```diff
@@ -0,0 +1,88 @@
+# Define the number of attributes allows per line (max-attributes-per-line)
+
+Limits the maximum number of props per line to improve readability
```

Comment:
I'd say `maximum number of attributes/properties` and add `.` at the end of the sentence.